norduser/service/combined.go

```diff
@@ -0,0 +1,86 @@
+package service
+
+import (
+	"fmt"
+	"log"
+	"sync"
+)
+
+type processType int
+
+const (
+	systemd processType = iota
+	fork
+)
+
+type Combined struct {
+	mu               sync.Mutex
+	systemd          SystemdNorduser
+	fork             ForkNorduser
+	uidToProcessType map[uint32]processType
```

Comment:
I assume this is designed to support both systemd and non systemd at the same time. Any valid reason for such environment?